examples/kv_addr_book/include/kv_addr_book.hpp

```diff
@@ -1,12 +1,13 @@
 #include <eosio/eosio.hpp>
+#include <eosio/table.hpp>
 
 // this structure defines the data stored in the kv::table
 struct person {
    eosio::name account_name;
-   eosio::non_unique<eosio::name, std::string> first_name;
-   eosio::non_unique<eosio::name, std::string> last_name;
-   eosio::non_unique<eosio::name, std::string, std::string, std::string, std::string> street_city_state_cntry;
-   eosio::non_unique<eosio::name, std::string> personal_id;
+   std::tuple<eosio::name, std::string> first_name;
+   std::tuple<eosio::name, std::string> last_name;
+   std::tuple<eosio::name, std::string, std::string, std::string, std::string> street_city_state_cntry;
+   std::tuple<eosio::name, std::string> personal_id;
```

Comment:
Can you remove all the references to non_unique in the comments below (lines 17, 21, 25, ...)?